server/routes/checkYourAnswers.js

```diff
@@ -38,8 +47,11 @@ module.exports = function CheckAnswerRoutes({ reportService, offenderService, in
       ]
 
       const data = reportSummary(form, offenderDetail, locationDescription, involvedStaff, incidentDate)
-
-      return res.render('pages/check-your-answers', { data, bookingId })
+      const prison = await locationService.getPrisonById(token, prisonId).catch(err => {
+        logger.warn(err, 'Prison details not found in elite-2')
+        return { description: getIn(['assignedLivingUnit', 'agencyName'], offenderDetail) }
```

Comment:
I don't think you should be handling the error here. 

This should never happen and we shouldn't handle things thats will reasonably never happen.
 
If it does happen this could potentially show the wrong prison which is probably worse than blowing up.

Reply:
also would suggest not using Ramda unless strictly necessary as it obfuscates intent around null handling and removes any type safety

Reply:
Done. I thought about not having the error checking but included it as a fallback.